ODIN_II/SRC/include/odin_types.h

```diff
@@ -315,6 +315,7 @@ enum operation_list {
     SPRAM,               // representing primitive single port ram
     DPRAM,               // representing primitive dual port ram
     YMEM,                // $mem block memory generated by ysos, can have complete varaiable num of read and write plus clk for each port
+    YMEM2,               // $mem_v2 block memory generated by ysos, can have complete varaiable num of read and write plus clk for each port
```

Comment:
typo (twice: variable)
Explain YMEM vs. YMEM2

Reply:
Thanks for review. I added detailed explanation regarding YMEM and YMEM2 in BlockMemories.cpp where we resolve Yosys memory types.